src/main/java/study/step5/domain/car/Cars.java

```diff
@@ -0,0 +1,51 @@
+package study.step5.domain.car;
+
+import java.util.Iterator;
+import java.util.List;
+import java.util.function.Consumer;
+import java.util.stream.Collectors;
+
+import study.step5.domain.movestrategy.MoveStrategy;
+
+public class Cars implements Iterable<Car>{
```

Comment:
```suggestion
public class Cars implements Iterable<Car> {
```

윈도우 : ctrl + alt + L
맥OS : option + command + L 
위 단축키를 통해서 컨벤션을 적용시킬 수 있습니다~